runtime/doc/repeat.txt

```diff
@@ -515,16 +515,20 @@ You would now have these files under ~/.local/share/nvim/site:
 	pack/foo/start/foobar/syntax/some.vim
 	pack/foo/opt/foodebug/plugin/debugger.vim
 
+When runtime files are searched for, first all paths in 'runtimepath' are
```

Comment:
I'm thinking we will need to starting reference this in various places. It describes the Nvim "runtime path" but not the 'runtimepath' option.

```suggestion
*runtimepath*
When runtime files are searched for, first all paths in 'runtimepath' are
```